express_relay/sdk/js/src/index.ts

```diff
@@ -591,4 +609,139 @@ export class Client {
       return response.data;
     }
   }
+
+  /**
```

Comment:
Ideally we can provide multiple functions to help here:
constructPermissionInstruction
createSvmBid: calls the above and adds it to the provided tx and returns an BidSvm object. needs searcher private key for signing and assumes the latestBlockhash is already inside the tx